charts/enterprise/nextcloud/docs/installation-notes.md

```diff
@@ -45,3 +45,9 @@ The following configurations must be set during initially setup for nextcloud to
 ### Storage
 
 You can change the User Data Storage option to your preference here if you previously setup the proper dataset permissions. All other Storage should remain the default of PVC.
+
+### FAQs
+
+**What if I get stuck in a deploy-loop?**
```

Comment:
LoL, absolutely not.
This can literally mean ANYTHING

Reply:
Yeah maybe it's a user error after all

https://discord.com/channels/830763548678291466/1178422329425002627/1209894470418440342